### PR TITLE
Added a test feature to Image creation that allowes to copy media ins…

### PR DESCRIPTION
**Pull request by Aharonyn**
…tead of moving it via django settings. Fixed tests to work with this feature.

### CODE REVIEW

File: run_tests.py [hunk 1]
```diff
@@ -1,42 +1,16 @@
...
-        Path(settings.MEDIA_ROOT)
-        / "MRI/DICOM"
-        / TEST_PATIENT_FIELDS["uid"]
-        / TEST_DWI_SERIES_FIELDS["uid"]
-    )
-    media.rmdir()
-
-    while str(media_dwi) != settings.MEDIA_ROOT:
-        media_dwi.rmdir()
-        media_dwi = media_dwi.parent
```
Comment (ZviBaratz) [inline run_tests.py:32]: What do you think about:
```python
from django.core.exceptions import ImproperlyConfigured

def clean_media():
    media = Path(settings.MEDIA_ROOT)
    if 'tests' in media:
        shutil.rmtree(media)
    else:
        raise ImproperlyConfigured("...")
```
?

File: tests/models/test_patient.py [hunk 2]
```diff
@@ -1,14 +1,13 @@
 from django.test import TestCase
-from django_dicom.models import Series, Patient, Study, Image, Header, DataElement
-from django_dicom.models.values import PersonName
+from django_dicom.models import Series, Patient, Study, Image  # , Header, DataElement
+
+# from django_dicom.models.values import PersonName
```
Comment (ZviBaratz) [inline tests/models/test_patient.py:4]: This can be removed

File: run_tests.py (hunk 1 above)
Comment (ZviBaratz) [reply run_tests.py:32]: Actually, if I understand correctly, this function isn't at all used anymore, is it?
It's possible that it would be better to replace the `shutil.rmtree(settings.IMPORTED_PATH)` below with something similar to the propsed `clean_media()` above, just to try and prevent an edge case in which someone somehow runs this function in the real environment and deletes the entire media directory.

File: tests/models/test_patient.py (hunk 2 above)
Comment (Aharonyn) [reply tests/models/test_patient.py:4]: Agreed

File: run_tests.py (hunk 1 above)
Comment (Aharonyn) [reply run_tests.py:32]: I can't think of a situation where this sort of thing will be possible, but I have no problem with this change.
